### DISCUSSION TITLE
How can I remove the ability to add files from the gallery in the application when starting the camera?

### DISCUSSION THREAD

**Discussion by PachinoMax** [quasarframework/quasar#9074] [General - Components / Directives / etc]
Hello.
How can I remove the ability to add files from the gallery in the application when starting the camera?
So that users can only take pictures and not download files from the gallery.
Help me please

**Comment by ontwikkelfabriek** [comment]
If you are using the Cordova camera plugin you can use `Camera.PictureSourceType = 1 (CAMERA only)` .

See:
https://cordova.apache.org/docs/en/10.x/reference/cordova-plugin-camera/#camerapicturesourcetype--enum
